app/code/Magento/Store/Ui/Component/Listing/Column/Store.php

```diff
@@ -93,11 +93,9 @@ public function prepareDataSource(array $dataSource)
     protected function prepareItem(array $item)
     {
         $content = '';
-        if (!empty($item[$this->storeKey])) {
-            $origStores = $item[$this->storeKey];
-        }
-
-        if (empty($origStores)) {
+        $origStores = $item[$this->storeKey] ?? null;
+        
+        if (is_null($origStores)) {
```

Comment:
Avoid is_null and prefer trict equality $origStores === null

Reply:
You're right. Fixed it.